src/arrayMethodJoin.js

```diff
@@ -3,9 +3,24 @@
 /**
  * Implement method join
  */
+
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let str = '';
```

Comment:
What is str? 💩

Reply:
String. I don't know what else there might be besides the string

Reply:
Think more about it 🤓
You need to describe what this is. Just str not enough, I don't understand what it